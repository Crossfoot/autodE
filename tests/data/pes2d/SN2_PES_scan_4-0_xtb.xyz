6

F   -2.89260  -0.16833  0.05397  
Cl  1.37955   0.08259   -0.02112 
C   -0.39737  -0.02311  0.00731  
H   -0.72454  -0.60210  -0.84559 
H   -0.70216  -0.50625  0.92565  
H   -0.80583  0.97712   -0.04152 
