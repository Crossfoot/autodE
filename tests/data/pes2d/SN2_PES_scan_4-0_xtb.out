      -----------------------------------------------------------      
     |                   =====================                   |     
     |                           x T B                           |     
     |                   =====================                   |     
     |                         S. Grimme                         |     
     |          Mulliken Center for Theoretical Chemistry        |     
     |                    University of Bonn                     |     
      -----------------------------------------------------------      

   * xtb version 6.2.3 (830e466) compiled by 'ehlert@majestix' on 2020-03-16

   xtb is free software: you can redistribute it and/or modify it under
   the terms of the GNU Lesser General Public License as published by
   the Free Software Foundation, either version 3 of the License, or
   (at your option) any later version.
   
   xtb is distributed in the hope that it will be useful,
   but WITHOUT ANY WARRANTY; without even the implied warranty of
   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
   GNU Lesser General Public License for more details.
   
   Cite this work as:
   * S. Grimme, C. Bannwarth, P. Shushkov, J. Chem. Theory Comput., 2017,
     13, 1989-2009. DOI: 10.1021/acs.jctc.7b00118
   * C. Bannwarth, S. Ehlert and S. Grimme., J. Chem. Theory Comput., 2019,
     15, 1652-1671. DOI: 10.1021/acs.jctc.8b01176
   * P. Pracht, E. Caldeweyher, S. Ehlert, S. Grimme, ChemRxiv, 2019, preprint.
     DOI: 10.26434/chemrxiv.8326202.v1
   
   for DFT-D4:
   * E. Caldeweyher, C. Bannwarth and S. Grimme, J. Chem. Phys., 2017,
     147, 034112. DOI: 10.1063/1.4993215
   * E. Caldeweyher, S. Ehlert, A. Hansen, H. Neugebauer, S. Spicher,
     C. Bannwarth and S. Grimme, J. Chem. Phys., 2019, 150, 154122.
     DOI: 10.1063/1.5090222
   
   for sTDA-xTB:
   * S. Grimme and C. Bannwarth, J. Chem. Phys., 2016, 145, 054103.
     DOI: 10.1063/1.4959605
   
   in the mass-spec context:
   * V. Asgeirsson, C. Bauer and S. Grimme, Chem. Sci., 2017, 8, 4879.
     DOI: 10.1039/c7sc00601b
   
   for metadynamics refer to:
   * S. Grimme, J. Chem. Theory Comput., 2019, 155, 2847-2862
     DOI: 10.1021/acs.jctc.9b00143
   
   with help from (in alphabetical order)
   C. Bannwarth, F. Bohle, G. Brandenburg, E. Caldeweyher, M. Checinski,
   S. Dohm, S. Ehlert, S. Ehrlich, F. März, H. Neugebauer, J. Pisarek,
   P. Pracht, P. Shushkov, and S. Spicher.
   
 * started run on 2020/04/08 at 23:04:01.656     

           -------------------------------------------------
          |                Calculation Setup                |
           -------------------------------------------------

          program call               : /home/tom/.local/bin/xtb SN2_PES_scan_4-0_xtb.xyz --chrg -1 --opt --input xcontrol_SN2_PES_scan_4-0
          coordinate file            : SN2_PES_scan_4-0_xtb.xyz
          omp threads                :                     1
          number of atoms            :                     6
          number of electrons        :                    22
          charge                     :                    -1
          spin                       :                   0.0
          first test random number   :      0.45443436020376

constraining bond 1 3 to    2.0000000 Å, actual value:    2.4998877 Å
constraining bond 2 3 to    1.7800000 Å, actual value:    1.7802880 Å

molecular fragmentation (1/2 indicates fragments):
122222
# atoms in fragment 1/2:     1     5
 fragment masses (1/2) :       19.00       50.49
CMA distance (Bohr)    :   7.047
constraining FC (au)   : 10.0000
########################################################################
[WARNING] Please study the warnings concerning your input carefully
-2- prog_main: XTBPATH is not set, using XTBHOME instead
-1- prog_main: XTBHOME is not set, using HOME instead
########################################################################
           ------------------------------------------------- 
          |                 G F N 2 - x T B                 |
          | Geometry, Frequencies, Noncovalent interactions |
          |            JCTC 2019 parametrisation            |
           ------------------------------------------------- 
             k(s)              :                1.8500
             k(p)              :                2.2300
             k(d)              :                2.2300
             k(f)              :                2.0000
             kEN (H0ij)        :               -2.0000
             D4 a1             :                0.5200
             D4 a2             :                5.0000
             D4 s6             :                1.0000
             D4 s8             :                2.7000
             D4 s9             :                5.0000
             alphaj            :                2.0000

  Z AO/shell   Hii/eV     exponent
  1     Wed Apr 25 08:07:45 CEST 2018   EN: 2.200 GM2: 0.406  GM3: 0.0800  RAES: 1.40
     1s    -10.707211    1.230000
  6     Wed Apr 25 08:07:45 CEST 2018   EN: 2.550 GM2: 0.538  GM3: 0.1500  RAES: 3.00
     2s    -13.970922    2.096432
     2p    -10.063292    1.800000
  9     Mon Apr 23 13:58:50 CEST 2018   EN: 3.980 GM2: 0.532  GM3: 0.1426  RAES: 2.40
     2s    -23.458179    2.416361
     2p    -15.746583    2.308399
 17     Mon Apr 23 14:09:22 CEST 2018   EN: 3.160 GM2: 0.249  GM3: 0.1495  RAES: 2.50
     3s    -29.278781    2.485265
     3p    -12.673758    2.199650
     3d     -0.240338    2.476089

           ------------------------------------------------- 
          |        Self-Consistent Charge Iterations        |
           ------------------------------------------------- 

          ...................................................
          :                      SETUP                      :
          :.................................................:
          :  # basis functions                  21          :
          :  # atomic orbitals                  20          :
          :  # shells                           10          :
          :  # electrons                        22          :
          :  max. iterations                   250          :
          :  Hamiltonian                  GFN2-xTB          :
          :  restarted?                      false          :
          :  GBSA solvation                  false          :
          :  PC potential                    false          :
          :  electronic temp.          300.0000000     K    :
          :  accuracy                    1.0000000          :
          :  -> integral cutoff          0.2500000E+02      :
          :  -> integral neglect         0.1000000E-07      :
          :  -> SCF convergence          0.1000000E-05 Eh   :
          :  -> wf. convergence          0.1000000E-03 e    :
          :  Broyden damping             0.4000000          :
          ...................................................

 iter      E             dE          RMSdq      gap      omega  full diag
   1    -13.1413178 -0.131413E+02  0.579E+00    5.41       0.0  T
   2    -13.1993328 -0.580150E-01  0.313E+00    3.05       1.0  T
   3    -13.2009910 -0.165816E-02  0.161E+00    2.46       1.0  T
   4    -13.2015783 -0.587367E-03  0.456E-01    1.87       1.0  T
   5    -13.2018271 -0.248752E-03  0.301E-01    2.23       1.0  T
   6    -13.2020405 -0.213392E-03  0.241E-01    2.03       1.0  T
   7    -13.2022401 -0.199622E-03  0.430E-02    2.00       1.0  T
   8    -13.2022420 -0.188225E-05  0.186E-02    1.97       2.2  T
   9    -13.2022428 -0.872721E-06  0.445E-03    1.97       9.2  T
  10    -13.2022429 -0.929751E-07  0.637E-04    1.98      64.0  T
  11    -13.2022429 -0.152865E-08  0.240E-04    1.98     170.1  T

   *** convergence criteria satisfied after 11 iterations ***

         #    Occupation            Energy/Eh            Energy/eV
      -------------------------------------------------------------
         1        2.0000           -0.9148349             -24.8939
       ...           ...                  ...                  ...
         5        2.0000           -0.3457720              -9.4089
         6        2.0000           -0.3355331              -9.1303
         7        2.0000           -0.2720811              -7.4037
         8        2.0000           -0.2720693              -7.4034
         9        2.0000           -0.0836459              -2.2761
        10        2.0000           -0.0819050              -2.2287
        11        2.0000           -0.0819036              -2.2287 (HOMO)
        12                         -0.0091876              -0.2500 (LUMO)
        13                          0.1693317               4.6078
        14                          0.1693362               4.6079
        15                          0.1720985               4.6830
        16                          0.1721021               4.6831
       ...                                ...                  ...
        20                          0.3858750              10.5002
      -------------------------------------------------------------
                  HL-Gap            0.0727160 Eh            1.9787 eV
             Fermi-level           -0.0455456 Eh           -1.2394 eV

 SCC (total)                   0 d,  0 h,  0 min,  0.006 sec
 SCC setup                      ...        0 min,  0.000 sec (  1.461%)
 Dispersion                     ...        0 min,  0.000 sec (  0.586%)
 integral evaluation            ...        0 min,  0.001 sec ( 11.776%)
 zeroth order Hamiltonian       ...        0 min,  0.000 sec (  0.285%)
 iterations                     ...        0 min,  0.003 sec ( 56.434%)
 molecular gradient             ...        0 min,  0.001 sec ( 22.329%)
 printout                       ...        0 min,  0.000 sec (  5.969%)

         :::::::::::::::::::::::::::::::::::::::::::::::::::::
         ::                     SUMMARY                     ::
         :::::::::::::::::::::::::::::::::::::::::::::::::::::
         :: total energy              -4.220699966314 Eh    ::
         :: gradient norm             26.724952858573 Eh/a0 ::
         :: HOMO-LUMO gap              1.978702416486 eV    ::
         ::.................................................::
         :: SCC energy               -13.202242939011 Eh    ::
         :: -> isotropic ES            0.250214366754 Eh    ::
         :: -> anisotropic ES         -0.003118186509 Eh    ::
         :: -> anisotropic XC         -0.002370901914 Eh    ::
         :: -> dispersion             -0.002049026432 Eh    ::
         :: repulsion energy           0.057887883366 Eh    ::
         :: add. restraining           8.923655260784 Eh    ::
         :::::::::::::::::::::::::::::::::::::::::::::::::::::

      ----------------------------------------------------------- 
     |                   =====================                   |
     |                        A N C O P T                        |
     |                   =====================                   |
     |               Approximate Normal Coordinate               |
     |                Rational Function Optimizer                |
      ----------------------------------------------------------- 

          ...................................................
          :                      SETUP                      :
          :.................................................:
          :   optimization level            normal          :
          :   max. optcycles                   200          :
          :   ANC micro-cycles                  20          :
          :   degrees of freedom                12          :
          :.................................................:
          :   RF solver                   davidson          :
          :   write xtbopt.log                true          :
          :   linear?                        false          :
          :   energy convergence         0.5000000E-05 Eh   :
          :   grad. convergence          0.1000000E-02 Eh/α :
          :   maximium RF displ.         1.0000000          :
          :   Hlow (freq-cutoff)         0.1000000E-01      :
          :   Hmax (freq-cutoff)         5.0000000          :
          :   S6 in model hess.         20.0000000          :
          ...................................................

generating ANC from model Hessian ...
Using Lindh-Hessian (1995)
 Shifting diagonal of input Hessian by   0.000000000000000E+000
 Lowest  eigenvalues of input Hessian
   -0.000000   -0.000000    0.000000    0.000000    0.000000    0.000000
    0.068517    0.068594    0.111792    0.111809    0.128418    0.483760
    0.575505    0.575536    6.271682    6.272745   20.213231   60.873756
 Highest eigenvalues
    0.575505    0.575536    6.271682    6.272745   20.213231   60.873756
 

........................................................................
.............................. CYCLE    1 ..............................
........................................................................
   1    -13.2022429 -0.132022E+02  0.152E-05    1.98       0.0  T
   2    -13.2022429 -0.198952E-12  0.121E-05    1.98    3376.9  T
   3    -13.2022429 -0.200728E-12  0.699E-06    1.98    5837.3  T
     SCC iter.                  ...        0 min,  0.000 sec
     gradient                   ...        0 min,  0.001 sec
 * total energy  :    -4.2207000 Eh     change       -0.1479232E-07 Eh
   gradient norm :    26.7249575 Eh/α   predicted     0.0000000E+00 (-100.00%)
   displ. norm   :     0.9108221 α      lambda       -0.2434167E+02
   maximum displ.:     0.7846388 α      in ANC's #9, #12, #6, ...

........................................................................
.............................. CYCLE    2 ..............................
........................................................................
   1    -13.0837414 -0.130837E+02  0.565E+00    2.65       0.0  T
   2    -13.2052818 -0.121540E+00  0.267E+00    5.67       1.0  T
   3    -13.2325207 -0.272389E-01  0.823E-01    4.80       1.0  T
   4    -13.2329280 -0.407238E-03  0.533E-01    4.77       1.0  T
   5    -13.2331448 -0.216802E-03  0.278E-01    4.84       1.0  T
   6    -13.2332844 -0.139617E-03  0.173E-01    4.79       1.0  T
   7    -13.2333668 -0.824364E-04  0.785E-02    4.87       1.0  T
   8    -13.2333838 -0.169452E-04  0.122E-02    4.86       3.4  T
   9    -13.2333840 -0.209419E-06  0.297E-03    4.85      13.8  T
  10    -13.2333840 -0.172300E-07  0.760E-04    4.85      53.7  T
     SCC iter.                  ...        0 min,  0.001 sec
     gradient                   ...        0 min,  0.001 sec
 * total energy  :    -7.8155967 Eh     change       -0.3594897E+01 Eh
   gradient norm :    24.5447477 Eh/α   predicted    -0.2226772E+02 ( 519.43%)
   displ. norm   :     0.6001942 α      lambda       -0.1184832E+02
   maximum displ.:     0.4310248 α      in ANC's #12, #9, #5, ...

........................................................................
.............................. CYCLE    3 ..............................
........................................................................
   1    -13.1850713 -0.131851E+02  0.261E+00    8.31       0.0  T
   2    -13.1934198 -0.834843E-02  0.152E+00    6.48       1.0  T
   3    -13.1974371 -0.401737E-02  0.859E-01    4.92       1.0  T
   4    -13.1980869 -0.649741E-03  0.578E-01    5.16       1.0  T
   5    -13.1989337 -0.846866E-03  0.321E-01    4.82       1.0  T
   6    -13.1991681 -0.234327E-03  0.156E-01    5.06       1.0  T
   7    -13.1992282 -0.601280E-04  0.308E-02    4.95       1.3  T
   8    -13.1992287 -0.529530E-06  0.530E-03    4.94       7.7  T
   9    -13.1992287 -0.321012E-07  0.128E-03    4.93      32.0  T
     SCC iter.                  ...        0 min,  0.001 sec
     gradient                   ...        0 min,  0.001 sec
 * total energy  :   -12.6231986 Eh     change       -0.4807602E+01 Eh
   gradient norm :     4.5815476 Eh/α   predicted    -0.8058240E+01 (  67.61%)
   displ. norm   :     0.4020725 α      lambda       -0.1759284E+01
   maximum displ.:     0.3972398 α      in ANC's #12, #9, #6, ...

........................................................................
.............................. CYCLE    4 ..............................
........................................................................
   1    -13.2097029 -0.132097E+02  0.946E-01    3.38       0.0  T
   2    -13.2109688 -0.126598E-02  0.637E-01    3.53       1.0  T
   3    -13.2111919 -0.223022E-03  0.444E-01    3.47       1.0  T
   4    -13.2114830 -0.291170E-03  0.232E-01    3.48       1.0  T
   5    -13.2116543 -0.171262E-03  0.606E-02    3.54       1.0  T
   6    -13.2116630 -0.865867E-05  0.125E-02    3.53       3.3  T
   7    -13.2116630 -0.751288E-07  0.405E-03    3.53      10.1  T
   8    -13.2116631 -0.206182E-07  0.305E-04    3.53     133.8  T
     SCC iter.                  ...        0 min,  0.001 sec
     gradient                   ...        0 min,  0.001 sec
 * total energy  :   -12.6868249 Eh     change       -0.6362628E-01 Eh
   gradient norm :     5.4866916 Eh/α   predicted    -0.1021848E+01 (1506.02%)
   displ. norm   :     0.1814477 α      lambda       -0.8852605E+00
   maximum displ.:     0.1631189 α      in ANC's #12, #9, #6, ...

........................................................................
.............................. CYCLE    5 ..............................
........................................................................
   1    -13.2064932 -0.132065E+02  0.444E-01    3.99       0.0  T
   2    -13.2068477 -0.354449E-03  0.204E-01    4.18       1.0  T
   3    -13.2068812 -0.335023E-04  0.103E-01    4.27       1.0  T
   4    -13.2068913 -0.101427E-04  0.620E-02    4.28       1.0  T
   5    -13.2068977 -0.642745E-05  0.453E-02    4.27       1.0  T
   6    -13.2069042 -0.645621E-05  0.550E-03    4.26       7.4  T
   7    -13.2069042 -0.497271E-07  0.238E-03    4.26      17.1  T
   8    -13.2069043 -0.812740E-08  0.380E-04    4.26     107.5  T
     SCC iter.                  ...        0 min,  0.002 sec
     gradient                   ...        0 min,  0.001 sec
 * total energy  :   -13.1357895 Eh     change       -0.4489646E+00 Eh
   gradient norm :     0.3390827 Eh/α   predicted    -0.4572029E+00 (   1.83%)
   displ. norm   :     0.0731816 α      lambda       -0.5524840E-02
   maximum displ.:     0.0602481 α      in ANC's #6, #5, #12, ...

........................................................................
.............................. CYCLE    6 ..............................
........................................................................
   1    -13.2004999 -0.132005E+02  0.153E-01    4.34       0.0  T
   2    -13.2005312 -0.312725E-04  0.723E-02    4.27       1.0  T
   3    -13.2005337 -0.245436E-05  0.294E-02    4.25       1.4  T
   4    -13.2005339 -0.219303E-06  0.104E-02    4.24       3.9  T
   5    -13.2005340 -0.996992E-07  0.406E-03    4.25      10.1  T
   6    -13.2005340 -0.539975E-07  0.115E-03    4.25      35.4  T
     SCC iter.                  ...        0 min,  0.001 sec
     gradient                   ...        0 min,  0.001 sec
 * total energy  :   -13.1386628 Eh     change       -0.2873296E-02 Eh
   gradient norm :     0.0435852 Eh/α   predicted    -0.2777219E-02 (  -3.34%)
   displ. norm   :     0.0385405 α      lambda       -0.3103912E-03
   maximum displ.:     0.0329623 α      in ANC's #5, #6, #12, ...

........................................................................
.............................. CYCLE    7 ..............................
........................................................................
   1    -13.1991433 -0.131991E+02  0.829E-02    4.24       0.0  T
   2    -13.1991461 -0.285899E-05  0.490E-02    4.23       1.0  T
   3    -13.1991464 -0.317314E-06  0.111E-02    4.22       3.7  T
   4    -13.1991466 -0.145234E-06  0.655E-03    4.22       6.2  T
   5    -13.1991466 -0.553203E-07  0.507E-03    4.22       8.1  T
   6    -13.1991467 -0.859720E-07  0.580E-04    4.22      70.4  T
     SCC iter.                  ...        0 min,  0.000 sec
     gradient                   ...        0 min,  0.001 sec
 * total energy  :   -13.1388545 Eh     change       -0.1916486E-03 Eh
   gradient norm :     0.0146571 Eh/α   predicted    -0.1554283E-03 ( -18.90%)
   displ. norm   :     0.0103351 α      lambda       -0.2459666E-04
   maximum displ.:     0.0082575 α      in ANC's #5, #6, #12, ...

........................................................................
.............................. CYCLE    8 ..............................
........................................................................
   1    -13.1986161 -0.131986E+02  0.218E-02    4.22       0.0  T
   2    -13.1986163 -0.177169E-06  0.132E-02    4.22       3.1  T
   3    -13.1986163 -0.255158E-07  0.310E-03    4.21      13.2  T
   4    -13.1986163 -0.125211E-07  0.202E-03    4.22      20.2  T
   5    -13.1986163 -0.555408E-08  0.158E-03    4.22      25.8  T
     SCC iter.                  ...        0 min,  0.001 sec
     gradient                   ...        0 min,  0.001 sec
 * total energy  :   -13.1388666 Eh     change       -0.1208809E-04 Eh
   gradient norm :     0.0002606 Eh/α   predicted    -0.1229980E-04 (   1.75%)
   displ. norm   :     0.0001775 α      lambda       -0.1458397E-07
   maximum displ.:     0.0001410 α      in ANC's #6, #8, #11, ...

........................................................................
.............................. CYCLE    9 ..............................
........................................................................
   1    -13.1986342 -0.131986E+02  0.118E-03    4.22       0.0  T
   2    -13.1986342 -0.873984E-09  0.938E-04    4.22      43.5  T
   3    -13.1986342 -0.177449E-08  0.485E-04    4.22      84.1  T
     SCC iter.                  ...        0 min,  0.001 sec
     gradient                   ...        0 min,  0.001 sec
 * total energy  :   -13.1388667 Eh     change       -0.1127139E-06 Eh
   gradient norm :     0.0002396 Eh/α   predicted    -0.7579836E-08 ( -93.28%)
   displ. norm   :     0.0008036 α      lambda       -0.7412916E-07
   maximum displ.:     0.0004617 α      in ANC's #8, #11, #6, ...

   *** GEOMETRY OPTIMIZATION CONVERGED AFTER 9 ITERATIONS ***

------------------------------------------------------------------------
 total energy gain   :        -8.9181667 Eh    -5596.2341 kcal/mol
 total RMSD          :         0.3213985 a0        0.1701 Å
 total power (kW/mol):     -2601.6270543 (step)********** (real)
------------------------------------------------------------------------

 ANCopt (total)                0 d,  0 h,  0 min,  0.027 sec
 optimizer setup                ...        0 min,  0.000 sec (  0.909%)
 model hessian                  ...        0 min,  0.000 sec (  1.095%)
 ANC generation                 ...        0 min,  0.000 sec (  1.245%)
 coordinate transformation      ...        0 min,  0.000 sec (  0.470%)
 single point calculation       ...        0 min,  0.025 sec ( 89.962%)
 optimization log               ...        0 min,  0.000 sec (  1.795%)
 hessian update                 ...        0 min,  0.000 sec (  0.258%)
 rational function              ...        0 min,  0.001 sec (  2.440%)

================
 final structure:
================
6
 xtb: 6.2.3 (830e466)
F        -2.52957739192917   -0.14748207096836    0.04659179522927
Cl        1.24516131107775    0.07463297132237   -0.01888943661529
C        -0.53269145542378   -0.03102350739631    0.00975760597817
H        -0.75602130270329   -0.61732511007376   -0.86547847989491
H        -0.73171072700847   -0.51919900888450    0.94874335159534
H        -0.83811043401303    1.00031672600055   -0.04202483629257
 
 Bond Distances (Angstroems)
 ---------------------------
Cl2-C3=1.7812        C3-Cl2=1.7812        C3-H4=1.0769         C3-H5=1.0769         C3-H6=1.0769         H4-C3=1.0769         
H5-C3=1.0769         H6-C3=1.0769         
 C  H  Rav=1.0769 sigma=0.0000  Rmin=1.0769  Rmax=1.0769     3
 Cl C  Rav=1.7812 sigma=0.0000  Rmin=1.7812  Rmax=1.7812     1
 
 selected bond angles (degree)
 --------------------
H4-C3-Cl2=103.07               H5-C3-Cl2=103.03               H5-C3-H4=115.06                H6-C3-Cl2=103.03               
H6-C3-H4=115.06                H6-C3-H5=115.07                
 
 selected dihedral angles (degree)
 ---------------------------------

 
 CMA Distance (Angstroems)
 ---------------------------
R(CMA):  3.2369
           -------------------------------------------------
          |                Final Singlepoint                |
           -------------------------------------------------

          ...................................................
          :                      SETUP                      :
          :.................................................:
          :  # basis functions                  21          :
          :  # atomic orbitals                  20          :
          :  # shells                           10          :
          :  # electrons                        22          :
          :  max. iterations                   250          :
          :  Hamiltonian                  GFN2-xTB          :
          :  restarted?                      false          :
          :  GBSA solvation                  false          :
          :  PC potential                    false          :
          :  electronic temp.          300.0000000     K    :
          :  accuracy                    1.0000000          :
          :  -> integral cutoff          0.2500000E+02      :
          :  -> integral neglect         0.1000000E-07      :
          :  -> SCF convergence          0.1000000E-05 Eh   :
          :  -> wf. convergence          0.1000000E-03 e    :
          :  Broyden damping             0.4000000          :
          ...................................................

 iter      E             dE          RMSdq      gap      omega  full diag
   1    -13.1986343 -0.131986E+02  0.325E-04    4.22       0.0  T
   2    -13.1986343 -0.874785E-10  0.277E-04    4.22     147.6  T
   3    -13.1986343 -0.184908E-09  0.146E-04    4.22     279.2  T

   *** convergence criteria satisfied after 3 iterations ***

         #    Occupation            Energy/Eh            Energy/eV
      -------------------------------------------------------------
         1        2.0000           -0.8911483             -24.2494
       ...           ...                  ...                  ...
         5        2.0000           -0.3294245              -8.9641
         6        2.0000           -0.3057421              -8.3197
         7        2.0000           -0.2487894              -6.7699
         8        2.0000           -0.2487806              -6.7697
         9        2.0000           -0.1247420              -3.3944
        10        2.0000           -0.1247353              -3.3942
        11        2.0000           -0.1243533              -3.3838 (HOMO)
        12                          0.0306023               0.8327 (LUMO)
        13                          0.2008616               5.4657
        14                          0.2008639               5.4658
        15                          0.2048433               5.5741
        16                          0.2048582               5.5745
       ...                                ...                  ...
        20                          0.4275296              11.6337
      -------------------------------------------------------------
                  HL-Gap            0.1549557 Eh            4.2166 eV
             Fermi-level           -0.0468755 Eh           -1.2755 eV

 SCC (total)                   0 d,  0 h,  0 min,  0.004 sec
 SCC setup                      ...        0 min,  0.000 sec (  1.391%)
 Dispersion                     ...        0 min,  0.000 sec (  0.246%)
 integral evaluation            ...        0 min,  0.001 sec ( 18.036%)
 zeroth order Hamiltonian       ...        0 min,  0.000 sec (  0.619%)
 iterations                     ...        0 min,  0.002 sec ( 43.535%)
 molecular gradient             ...        0 min,  0.001 sec ( 29.268%)
 printout                       ...        0 min,  0.000 sec (  4.860%)

         :::::::::::::::::::::::::::::::::::::::::::::::::::::
         ::                     SUMMARY                     ::
         :::::::::::::::::::::::::::::::::::::::::::::::::::::
         :: total energy             -13.138866655722 Eh    ::
         :: gradient norm              0.000242217579 Eh/a0 ::
         :: HOMO-LUMO gap              4.216558411586 eV    ::
         ::.................................................::
         :: SCC energy               -13.198634250641 Eh    ::
         :: -> isotropic ES            0.219621845796 Eh    ::
         :: -> anisotropic ES         -0.007967412809 Eh    ::
         :: -> anisotropic XC         -0.010563867326 Eh    ::
         :: -> dispersion             -0.002201509884 Eh    ::
         :: repulsion energy           0.059700829090 Eh    ::
         :: add. restraining           0.000066788101 Eh    ::
         :::::::::::::::::::::::::::::::::::::::::::::::::::::

           -------------------------------------------------
          |                Property Printout                |
           -------------------------------------------------

    * Orbital Energies and Occupations

         #    Occupation            Energy/Eh            Energy/eV
      -------------------------------------------------------------
         1        2.0000           -0.8911483             -24.2494
         2        2.0000           -0.4278050             -11.6412
         3        2.0000           -0.3960223             -10.7763
         4        2.0000           -0.3294514              -8.9648
         5        2.0000           -0.3294245              -8.9641
         6        2.0000           -0.3057421              -8.3197
         7        2.0000           -0.2487894              -6.7699
         8        2.0000           -0.2487806              -6.7697
         9        2.0000           -0.1247420              -3.3944
        10        2.0000           -0.1247353              -3.3942
        11        2.0000           -0.1243533              -3.3838 (HOMO)
        12                          0.0306023               0.8327 (LUMO)
        13                          0.2008616               5.4657
        14                          0.2008639               5.4658
        15                          0.2048433               5.5741
        16                          0.2048582               5.5745
        17                          0.2810511               7.6478
        18                          0.3662735               9.9668
        19                          0.4273963              11.6300
        20                          0.4275296              11.6337
      -------------------------------------------------------------
                  HL-Gap            0.1549557 Eh            4.2166 eV
             Fermi-level           -0.0468755 Eh           -1.2755 eV

     #   Z        covCN         q      C6AA      α(0)
     1   9 F      0.157    -0.834    20.937     5.333
     2  17 Cl     0.889    -0.414   112.641    16.237
     3   6 C      3.822     0.076    19.294     6.251
     4   1 H      0.926     0.057     2.234     2.337
     5   1 H      0.926     0.057     2.229     2.335
     6   1 H      0.926     0.057     2.230     2.335

 Mol. C6AA /au·bohr⁶  :        572.118972
 Mol. C8AA /au·bohr⁸  :      15892.757132
 Mol. α(0) /au        :         34.827289


Wiberg/Mayer (AO) data.
largest (>0.10) Wiberg bond orders for each atom
          total WBO             WBO to atom ...
     1  F   0.304        C    3 0.198    Cl   2 0.105
     2  Cl  0.869        C    3 0.749    F    1 0.105
     3  C   3.920        H    4 0.991    H    6 0.991    H    5 0.991    Cl   2 0.749    F    1 0.198
     4  H   0.997        C    3 0.991
     5  H   0.997        C    3 0.991
     6  H   0.997        C    3 0.991

molecular dipole:
                 x           y           z       tot (Debye)
 q only:        2.686       0.155      -0.052
   full:        2.037       0.117      -0.042       5.187
molecular quadrupole (traceless):
                xx          xy          yy          xz          yz          zz
 q only:      -21.207      -1.865      10.555       0.574       0.032      10.653
  q+dip:      -26.219      -2.312      13.048       0.698       0.040      13.171
   full:      -24.813      -2.186      12.348       0.663       0.038      12.464

           -------------------------------------------------
          |                Geometry Summary                 |
           -------------------------------------------------

      molecular mass/u    :       69.4858989
   center of mass at/Å    :       -0.1821324      -0.0095828       0.0053859
  moments of inertia/u·Å² :        0.3327978E+01   0.1817937E+03   0.1817938E+03
rotational constants/cm⁻¹ :        0.5065427E+01   0.9272948E-01   0.9272941E-01

 * 4 selected distances

     #   Z        #   Z                                     value/Å
     2  17 Cl     3   6 C                                 1.7812199 (max)
     3   6 C      4   1 H                                 1.0768770
     3   6 C      5   1 H                                 1.0768557 (min)
     3   6 C      6   1 H                                 1.0768588

 * 2 distinct bonds (by element types)

   Z      Z             #   av. dist./Å        max./Å        min./Å
   1 H    6 C           3     1.0768638     1.0768770     1.0768557
   6 C   17 Cl          1     1.7812199     1.7812199     1.7812199


optimized geometry written to: xtbopt.xyz


           -------------------------------------------------
          | TOTAL ENERGY              -13.138866655722 Eh   |
          | GRADIENT NORM               0.000242217579 Eh/α |
          | HOMO-LUMO GAP               4.216558411586 eV   |
           -------------------------------------------------

------------------------------------------------------------------------
 * finished run on 2020/04/08 at 23:04:01.707     
------------------------------------------------------------------------
 total:
 * wall-time:     0 d,  0 h,  0 min,  0.051 sec
 *  cpu-time:     0 d,  0 h,  0 min,  0.049 sec
 * ratio c/w:     0.968 speedup
 SCF:
 * wall-time:     0 d,  0 h,  0 min,  0.006 sec
 *  cpu-time:     0 d,  0 h,  0 min,  0.006 sec
 * ratio c/w:     1.000 speedup
 ANC optimizer:
 * wall-time:     0 d,  0 h,  0 min,  0.032 sec
 *  cpu-time:     0 d,  0 h,  0 min,  0.032 sec
 * ratio c/w:     1.000 speedup

